website_sale_delivery_date/readme/CONTEXT.md

```diff
@@ -0,0 +1,49 @@
+This module extends Odoo's e-commerce checkout process by adding a delivery date selection tied to the chosen delivery method. The selected delivery date is then transferred to the created Sale Order's Delivery Date field (commitment_date).
+
+To support realistic and flexible delivery scheduling, the module introduces configurable constraints on delivery timing, including:
```

Comment:
This is irrelevant to the CONTEXT section. Please check [what is file is about](https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/CONTEXT.md) and update the documentation accordingly.